c7n/resources/sns.py

```diff
@@ -464,6 +464,37 @@ class resource_type(TypeInfo):
             'TopicArn'
         )
 
+@SNSSubscription.filter_registry.register('unused')
+class UnusedSNSSubscription(CrossAccountAccessFilter):
```

Comment:
I don't think we need to derive from the CrossAccountAccessFilter here - this may make more sense as a related resource filter (similar to how EBS snapshots have a [`volume`](https://github.com/cloud-custodian/cloud-custodian/blob/c7d66429d441619984294be043bc8bcc896c8c12/c7n/resources/ebs.py#L368-L396) filter).